src/main/java/lotto/model/LottoNumbers.java

```diff
@@ -0,0 +1,40 @@
+package lotto.model;
+
+import java.util.LinkedHashSet;
+import java.util.Set;
+import java.util.stream.Collectors;
+
+public class LottoNumbers {
+    public static final int LOTTO_MAX_COUNT = 6;
+    public static final int LOTTO_MIN_RANGE = 1;
+    public static final int LOTTO_MAX_RANGE = 45;
+    private final Set<Integer> lottoNumbers;
+
+    public LottoNumbers(Set<Integer> lottoNumbers) {
+        validateSizeAndDistinct(lottoNumbers);
+        validateNumberRange(lottoNumbers);
+        this.lottoNumbers = lottoNumbers.stream().sorted().collect(Collectors.toCollection(LinkedHashSet::new));
```

Comment:
메서드 체이닝을 할때는 행구분을 해주시는 것이 좋습니다.

```suggestion
        this.lottoNumbers = lottoNumbers.stream()
            .sorted()
            .collect(Collectors.toCollection(LinkedHashSet::new));
```